scripts/identify-web-features.js

```diff
@@ -0,0 +1,303 @@
+      }
+    }
+  }
+}
+
+// Identify web-features based on explorer URLs in the issue body.
+function gatherFeaturesFromExplorerUrls(issueBody, gatheredFeatures) {
+  // https://web-platform-dx.github.io/web-features-explorer/features/<feature-id>
+  let urls = issueBody.match(/https?:\/\/web-platform-dx.github.io\/web-features-explorer\/features\/[a-z0-9-]+/g) || [];
+  urls = urls.map(url => url.split("#")[0]);
+
+  for (const url of urls) {
+    // Extract the feature ID from the URL.
+    const match = url.match(/features\/([a-z0-9-]+)/);
+    if (match && match[1]) {
+      const id = match[1];
+      const feature = features[id];
+      if (feature) {
+        gatheredFeatures.add(id);
+      }
+    }
+  }
+}
+
+// Identify web-features based on WPT URLs in the issue body.
+function gatherFeaturesFromWPTUrls(issueBody, gatheredFeatures) {
+  // https://wpt.fyi/results/?q=feature:<feature-id>
+  let urls = issueBody.match(/https?:\/\/wpt\.fyi\/results\/\?q=feature:[a-z0-9-]+/g) || [];
+  urls = urls.map(url => url.split("#")[0]);
+
+  for (const url of urls) {
+    // Extract the feature ID from the URL.
+    const match = url.match(/feature:([a-z0-9-]+)/);
+    if (match && match[1]) {
+      const id = match[1];
+      const feature = features[id];
+      if (feature) {
+        gatheredFeatures.add(id);
+      }
+    }
+  }
+}
+
+// Given a GitHub issue, find the web-features that are referenced in the issue body.
+// For now, we only look for URLs that match the web-features specifications.
+function findFeaturesInIssue(issue) {
+  const features = new Set();
+  
+  gatherFeaturesFromSpecUrls(issue.body, features);
+  gatherFeaturesFromExplorerUrls(issue.body, features);
+  gatherFeaturesFromWPTUrls(issue.body, features);
+  // TODO: Match on other things too:
+  // - web-features ID in the body of the issue, which might be there since the issue template asks for it.
+  // - MDN URLs
+  // - vendor positions
+  // - etc.
+  // Maybe investigate a weighting system to prioritize features based on the number of matches.
+
+  return [...features];
+}
+
+// Given a feature id, retrieve the feature's data.
+// We use the web-features-explorer's JSON files to get the full data, which includes both
+// the data that comes from the web-features project and the additional data that the explorer augments it with.
+async function getFeatureData(id) {
+  console.log(`Getting data for feature ${id}`);
+
+  try {
+    const response = await fetch(`https://web-platform-dx.github.io/web-features-explorer/features/${id}.json`);
+    return await response.json();
+  } catch (error) {
+    console.error(`Error fetching the feature data for ${id}:`, error);
+    return null;
+  }
+}
+
+function printBaselineStatus(feature) {
+  if (feature.status && feature.status.baseline === "high") {
+    return "Widely Available";
+  } else if (feature.status && feature.status.baseline === "low") {
+    return "Newly Available";
+  }
+  return "Limited Availability";
```

Comment:
There might also be totally unimplemented features, right?

Reply:
Yes, but Limited Availability currently encompasses them too.